app/views/view_data/_field_message.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if field_error?(settings, [field]) %>
+  <ul class="error-message">
+    <% settings.errors.messages.each do |k, v| %>
+      <% v.each do |value| %>
+        <li><%= "#{k.to_s.humanize} #{value}" if k == field %><li>
```

Comment:
One tiny suggested change, how about ```#{k.to_s.humanize} #{value}``` to ```#{k.to_s.humanize}: #{value}```

The : will hopefully separate the field name from the message.